src/Sylius/Bundle/CoreBundle/Form/Type/CustomerGuestType.php

```diff
@@ -9,10 +9,9 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Bundle\UserBundle\Form\Type;
+namespace Sylius\Bundle\CoreBundle\Form\Type;
```

Comment:
It uses `SetCustomerFormSubscriber`, so after [this comment](https://github.com/Sylius/Sylius/pull/5547#discussion_r71855046) it's clear we should leave it in `CoreBundle` :)